context-areas/ecosystem.md

```diff
@@ -0,0 +1,72 @@
+# Ecosystem
```

Comment:
This seems like a good addition to me, and provides some clarity vis a vi a useful example.